feature-libs/user/account/assets/translations/en/user-account.ts

```diff
@@ -25,4 +25,7 @@ export const userAccount = {
     userGreeting: 'Hi, {{name}}',
     signInRegister: 'Sign In / Register',
   },
+  MyAccountV2User: {
```

Comment:
[MEDIUM]
For consistency reasons, let's keep i18n translation keys pascalCase

Reply:
done